plugins/inputs/netdata/README.md

```diff
@@ -0,0 +1,64 @@
+# Netdata Input Plugin
+
+The [Netdata](https://github.com/firehol/netdata) plugin reads all metrics
+generated by a Netdata instance.
+
+### Configuration
+
+```toml
+  [[inputs.netdata]]
+  ## Number of points to gather.
+  ## By default inputs plugin run every 10 seconds, by asking 10 points
+  ## to netdata we get measurement for each second. This value is capped by "update every" config of netdata.
+  ## For example you can set
+  ## points = 20
+  ## interval = "10m"
+  ## to get measurement every 30 seconds (and polling netdata every 10 minutes)
+  ## default to 10
+  points = 10
+
+  ## Grouping method if multiple collected values are to be grouped in order to return fewer points.
+  ## methods supported "min", "max", "average", "sum", "incremental-sum"
+  ## cf : https://github.com/firehol/netdata/wiki/REST-API-v1
+  ## default to average
+  group = "average"
+
+  ## Should measurement be prefixed by the netdata hostname ?
+  ## Ex : netdata.host1.apps.cpu (with hostname),  netdata.apps.cpu (without hostname)
+  ## default to true
+  prefixChartIDByHostname = true
+
+  ## Netdata servers
+  [[inputs.netdata.servers]]
+    url="http://netdata-host:19999"
+
+  [[inputs.netdata.servers]]
+    url="https://netdata-host2"
+    basicAuthUsername="user"
+    basicAuthPassword="pass"
+```
+
+### Measurements & Fields:
+
+Measurements and Fields depend on charts generated by Netdata. Netdata exposes charts through its API : /api/v1/charts.
+Every chart generate a measurement with the following name : "netdata.\<hostname\>.\<netdata_chart_id\>" (hostname can be removed using the config key "prefixChartIDByHostname").
```

Comment:
Could you please make this less technical e.g. by giving an example and leave out the API details!?